ntbs-service/Pages/Shared/_NotificationOverviewLayout.cshtml

```diff
@@ -31,7 +31,19 @@
             @if (!string.IsNullOrEmpty(Model.PermissionReason))
             {
                 <div class="notification-overview-details-container no-print" >
-                    <nhs-inset-text classes="nhs-inset-text--ntbs">@Model.PermissionReason</nhs-inset-text>
+                    <nhs-inset-text classes="nhs-inset-text--ntbs">
+                        <span>
+                            @Model.PermissionReason
+                            @if (Model.PermissionReason == Messages.ClosedNoEdit)

```

Comment:
It seems that we would want to link to this FAQ on `Messages.NoEditPermission` as well, no?
There's also `Why can’t I view the full details of a record?` which would make sense to link from the `Messages.UnauthorizedWarning` permission? :)